homeassistant/components/zwave_js/config_flow.py

```diff
@@ -503,18 +501,6 @@ async def async_step_usb(self, discovery_info: UsbServiceInfo) -> ConfigFlowResu
             )
             title = human_name.split(" - ")[0].strip()
         self.context["title_placeholders"] = {CONF_NAME: title}
-        self._title = title
-        return await self.async_step_usb_confirm()
-
-    async def async_step_usb_confirm(
-        self, user_input: dict[str, Any] | None = None
-    ) -> ConfigFlowResult:
-        """Handle USB Discovery confirmation."""
-        if user_input is None:
-            return self.async_show_form(
-                step_id="usb_confirm",
-                description_placeholders={CONF_NAME: self._title},
-            )
 
         self._usb_discovery = True
         if current_config_entries := self._async_current_entries(include_ignore=False):
```

Comment:
We can set `self._reconfigure_config_entry` already above on line 477, and simplify here to just check the variables `current_config_entries` and `self._reconfigure_config_entry` to go to the migrate step or to the installation type step.